### ISSUE TITLE
How do I add enum to my GraphQLSchema?

### ISSUE THREAD

**Issue by westbranch** [leangen/graphql-spqr#44]
In my app I have this enum as a separate class:
```
public enum UserSortFields {
    ID,  
    USERNAME,
    EMAIL,
    VALID_FROM,
    ACCESS_PERIOD,
    MAX_SESSION_AMOUNT
}
```
I can't seem to understand how do I add this one when generating the GraphQLSchema?

**Comment by kaqqao** [comment]
If `UserSortFields` is the return type or a parameter type of any of the resolver methods, it will be mapped to a corresponding GraphQl enum type.

e.g. if you have:

```java
@GraphQLQuery
public List<User> findUsersByStreet(
                            @GraphQLArgument(name = "street") Street street,
                            @GraphQLArgument(name = "sortBy") List<UserSortFields> sortFields) {
    ....
}
```
the `sortBy` argument will be a list of `GraphQLEnum`s. Is that what you're after?

**Comment by kaqqao** [comment]
Did this answer you question? Or did you mean something else?

**Comment by westbranch** [comment]
Sorry for the delay. 
That didn't answer my question, but gave me an idea how to solve my problem. 
Basically, the idea was to make these enums accessible for the front-end via some graphql query. 
I listed all the enums in a separate class, then annotated that class with my custom annotation. Then I created a method which collects annotated beans to an ArrayList and then I add them to my GraphQLSchema.
